core/services/keeper/validate.go

```diff
@@ -79,9 +63,5 @@ func ValidatedKeeperSpec(tomlString string) (job.Job, error) {
 		return j, errors.Errorf("unsupported type %s", j.Type)
 	}
 
```

Comment:
i think we need to inject the Observation Source in here. I dont see where you are putting the Observation Source into the job spec

Reply:
@vla33 We'll won't ask the user for obs source but we will need to inject the hardcoded source into 'j' so that it can be used downstream